<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementsWidget</class>
 <widget class="QWidget" name="MeasurementsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="DateTimeFilterWidget" name="dateTimeFilter" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QCheckBox" name="useTemperature">
           <property name="text">
            <string>Temperature Filter</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/temperature.png</normaloff>:/icons/ui/temperature.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="minHumidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string>&gt;=  </string>
           </property>
           <property name="suffix">
            <string> %RH</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>99.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="minTemperature">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string>&gt;=  </string>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
           <property name="maximum">
            <double>199.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="maxHumidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string>&lt;= </string>
           </property>
           <property name="suffix">
            <string> %RH</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>70.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="useHumidity">
           <property name="text">
            <string>Humidity Filter</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/humidity.png</normaloff>:/icons/ui/humidity.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="maxTemperature">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string>&lt;= </string>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-49.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>30.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="showTemperature">
           <property name="text">
            <string>Show Temperature</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/temperature.png</normaloff>:/icons/ui/temperature.png</iconset>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="showHumidity">
           <property name="text">
            <string>Show Humidity</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/humidity.png</normaloff>:/icons/ui/humidity.png</iconset>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="showBattery">
           <property name="text">
            <string>Show  Battery</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/battery-75.png</normaloff>:/icons/ui/battery-75.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="showSignal">
           <property name="text">
            <string>Show Signal</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/signal-80.png</normaloff>:/icons/ui/signal-80.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="selectSensors">
           <property name="text">
            <string>Select Sensors</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/sensor.png</normaloff>:/icons/ui/sensor.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportData">
           <property name="text">
            <string>Export</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/export.png</normaloff>:/icons/ui/export.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="resultCount">
           <property name="text">
            <string>0 results</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="list">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateTimeFilterWidget</class>
   <extends>QWidget</extends>
   <header>DateTimeFilterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>useTemperature</tabstop>
  <tabstop>minTemperature</tabstop>
  <tabstop>maxTemperature</tabstop>
  <tabstop>useHumidity</tabstop>
  <tabstop>minHumidity</tabstop>
  <tabstop>maxHumidity</tabstop>
  <tabstop>showTemperature</tabstop>
  <tabstop>showHumidity</tabstop>
  <tabstop>showBattery</tabstop>
  <tabstop>showSignal</tabstop>
  <tabstop>selectSensors</tabstop>
  <tabstop>exportData</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useTemperature</sender>
   <signal>toggled(bool)</signal>
   <receiver>minTemperature</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTemperature</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTemperature</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useHumidity</sender>
   <signal>toggled(bool)</signal>
   <receiver>minHumidity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useHumidity</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxHumidity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
